src/frontend/cli.rs

```diff
@@ -462,6 +462,25 @@ pub fn main() {
                         &project_manifest.project.authors,
                     ),
                 ));
+
+                #[cfg(feature = "telemetry")]
```

Comment:
Should we make sure that developers enabled telemetry in their project before?
`if manifest.project.telemetry`

Reply:
See line 457:

```rust
            if project_manifest.project.telemetry {
```